### DISCUSSION TITLE
Auto jump to latest image added to the folder

### DISCUSSION THREAD

**Discussion by bertoo87** [d2phap/ImageGlass#1867] [General] [answered]
Hi there :)

Would it be possible to add a setting or toggle for ImageGlass to automatically jump to the latest image that was added to a folder?
Reson for why this would be awesome: My Camera transfers files wirelessly to my PC. When i have an image open of this folder in IG, i can see that the thumbnail in the gallery bar at the bottom is added, but i have to jump to it manually. Since its appearing there I assume this should be possible. This would be a game changer for me.

Greetings from the alps :)

**Comment by d2phap** [comment, marked as answer]
Hi @bertoo87,

You can check this option "Open the new added image automatically"

![image](https://github.com/d2phap/ImageGlass/assets/3154213/7c042b13-f937-4857-ad6f-3393195d7001)

**Reply by bertoo87** [reply]
damn! you can't imagine how embarrassed i feel right now... how the hell did i miss this.
In that case: Thanks for creating the best photo viewer software out there, a donation is coming your way as compensation for taking the time for that screenshot 😅

**Reply by d2phap** [reply]
Thanks for your generousness!
